pmm-app/src/pmm-update-panel/module.ts

```diff
@@ -197,30 +210,30 @@ export class PanelCtrl extends MetricsPanelCtrl {
      * Send request for get info about update status
      */
     private getLog($scope, $http): void {
-        if (!$scope.logLocation.length) return;
-
+        if ($scope.updateCntErrors > 600) {
+            console.log('error: ', $scope.errorMessage, $scope.updateCntErrors);
+            $scope.updateFailed = true;
+            return;
```

Comment:
Previous code also set canBeReloaded. I have no idea if that is needed :)

Reply:
I have added it here https://github.com/percona/grafana-dashboards/blob/322928e21cfbdaad227b7286bfda6dd21220552b/pmm-app/src/pmm-update-panel/module.ts#L219